Source/DataSources/Angle.js

```diff
@@ -0,0 +1,97 @@
+/*global define*/
+define([
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/DeveloperError',
+        '../Core/Event'
+    ], function(
+        defaultValue,
+        defined,
+        defineProperties,
+        DeveloperError,
+        Event) {
+    "use strict";
+
+    /**
+     * A property type whose value is always interpolated by taking the the shortest distances between 2 angles.
+
+     *
+     * @alias ConstantProperty
+     * @constructor
+     *
+     * @param {Object} [value] The property value.
+     *
+     * @see ConstantPositionProperty
+     *
+     * @exception {DeveloperError} value.clone is a required function.
+     * @exception {DeveloperError} value.equals is a required function.
+     */
+
+    //A value that always interpolates the shortest distances between 2 angles.
+    var Angle = {
```

Comment:
I'd prefer we rename this to Rotation instead of Angle, since the former is more specific.